src/Module.zig

```diff
@@ -3703,6 +3703,44 @@ fn semaDecl(mod: *Module, decl: *Decl) !bool {
         };
     };
 
+    if (mod.comp.verbose_air) {
```

Comment:
```suggestion
    if (builtin.mode == .Debug and mod.comp.verbose_air) {
```

This makes us not have to pay the cost of the code in this block for release builds